plugin.js

```diff
@@ -46,6 +49,10 @@ function plugin (fn, options = {}) {
     fn.default = fn
   }
 
+  if (!autoName && !fn[options.name]) {
+    fn[options.name] = fn
```

Comment:
If kebab-case is used for naming plugins, ex. `fastify-swagger`, makes the plugin non-importable.
However `-` is not allowed as var name. I think we need to transform kebab case in camelCase in order to be able to import like:
`import { fastifySwagger } from 'fastify-swagger'`

Reply:
Implemented